### PR TITLE
Add litecoin docker image and container.

### PR DESCRIPTION
**Pull request by AndrewSamokhvalov**

### CODE REVIEW

File: docker/lnd/start-lnd.sh [hunk 1]
```diff
@@ -42,16 +42,17 @@ set_default() {
...
-    --bitcoin.active \
-    "--bitcoin.$BITCOIN_NETWORK" \
-    --bitcoin.rpchost="btcd" \
-    --bitcoin.rpccert="/rpc/rpc.cert" \
-    --bitcoin.rpcuser="$RPCUSER" \
-    --bitcoin.rpcpass="$RPCPASS" \
+    "--$CHAIN.rpccert"="/rpc/rpc.cert" \
+    "--$CHAIN.active" \
+    "--$CHAIN.$NETWORK" \
+    "--$CHAIN.rpchost"="blockchain" \
```
Comment (AndrewSamokhvalov) [inline docker/lnd/start-lnd.sh]: Nope :)

File: docker/README.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ bob$ lncli listpeers
 Create the `Alice<->Bob` channel.
 ```bash
 # Open the channel with "Bob":
-alice$ lncli openchannel --node_key=<bob_identity_pubkey> --num_confs=1 --local_amt=1000000
+alice$ lncli openchannel --peer_id=1 --local_amt=1000000
```
Comment (Roasbeef) [inline docker/README.md]: The `peer_id` arguments are largely being phased out, why the change to identitfy by the peer ID? As is now, users won't be notified of the peer ID of a peer upon connection.

Comment (Roasbeef) [inline docker/README.md]: The `peer_id` arguments are largely being phased out, why the change to identitfy by the peer ID? As is now, users won't be notified of the peer ID of a peer upon connection.

File: docker/README.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ Connect `Bob` node to `Alice` node.
 
 ```bash
 # Run "Bob" node and log into it:
-$ docker-compose up --no-recreate -d "bob"
+$ docker-compose up --no-recreate -d --name "bob" "ltc_btc"
```
Comment (Roasbeef) [inline docker/README.md]: The last argument should instead be `"lnd_btc"`.

File: docker/README.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ bitcoin into.
 $ export NETWORK="simnet" 
 
 # Run the "Alice" container and log into it:
-$ docker-compose up -d "alice"
+$ docker-compose run -d --name alice "lnd_btc"
```
Comment (Roasbeef) [inline docker/README.md]: Quotes aren't needed around the `"lnd_btc"` argument, here or below.

File: docker/lnd/start-lnd.sh (hunk 1 above)
Comment (Roasbeef) [inline docker/lnd/start-lnd.sh]: Tested this locally, and looks like it does indeed to to be scope somehow. 

Run from inside the container:
```
root@aec17b768d6e:/go/src/github.com/lightningnetwork/lnd# ps xa
  PID TTY      STAT   TIME COMMAND
    1 ?        Ss     0:00 bash ./start-lnd.sh
   11 ?        Sl     0:04 lnd --datadir=/data --logdir=/data --bitcoin.rpccert=/rpc/rpc.cert --bitcoin.active --bitcoin.simnet --bitcoin.rpchost=blockchain --bitcoin.rpcuser=devuser --bitcoin.rpcpass=devpass --d
   27 pts/0    Ss     0:00 bash
   33 pts/0    R+     0:00 ps xa
```
The point of interest is: `--bitcoin.rpchost=blockchain`, which means the string was passed through without any sort of substitution.

Comment (Roasbeef) [inline docker/lnd/start-lnd.sh]: Actually scratch that. Not sure what happened before, but I'm able to properly connect now! Once the minor doc comments are addressed, I'll get this merged.
